packages/blocks/src/embed-linked-doc-block/embed-linked-doc-block.ts

```diff
@@ -445,6 +456,20 @@ export class EmbedLinkedDocBlockComponent extends EmbedBlockComponent<EmbedLinke
     );
   }
 
+  resetState(): {
+    loading?: boolean;
```

Comment:
Doesn't it need any parameters?

Reply:
It seems that you can directly return an empty object